packages/block-editor/src/components/block-lock/modal.js

```diff
@@ -171,6 +194,16 @@ export default function BlockLockModal( { clientId, onClose } ) {
 							/>
 						</li>
 					</ul>
+					{ hasTemplateLock && (
+						<ToggleControl
+							className="block-editor-block-lock-modal__template-lock"
+							label={ __( 'Apply to inner blocks' ) }
```

Comment:
"Inner blocks" is a bit of technical jargon. It'd be better to refer to it as "Apply to all nested blocks" or "Apply to all children".

Reply:
Good point. I like the simplicity of the "Apply to all children" label.